GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/QuizActivity.java

```diff
@@ -1,29 +1,72 @@
 package com.bignerdranch.android.geoquiz;
 
+import androidx.appcompat.app.AppCompatActivity;
+
 import android.os.Bundle;
 import android.view.Gravity;
+import android.view.View;
 import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TextView;
 import android.widget.Toast;
 
 import androidx.annotation.StringRes;
-import androidx.appcompat.app.AppCompatActivity;
 
 public class QuizActivity extends AppCompatActivity {
   @SuppressWarnings("FieldCanBeLocal")
   private Button trueButton;
   @SuppressWarnings("FieldCanBeLocal")
   private Button falseButton;
+  private ImageButton nextButton;
+  private ImageButton prevButton;
+  private TextView questionTextView;
+  private Question[] questionBank = new Question[] {
```

Comment:
I'd make a controller to provide this questions, something simple, just a singleton